packages/protocol/contracts/governance/GovernanceSlasher.sol

```diff
@@ -85,41 +67,12 @@ contract GovernanceSlasher is
    * @param electionIndices Indices of groups voted by slashed account.
    */
   function slash(
-    address account,
-    address[] calldata electionLessers,
-    address[] calldata electionGreaters,
-    uint256[] calldata electionIndices
-  ) external onlyL1 returns (bool) {
-    uint256 penalty = slashed[account];
-    require(penalty > 0, "No penalty given by governance");
-    slashed[account] = 0;
-    getLockedGold().slash(
-      account,
-      penalty,
-      address(0),
-      0,
-      electionLessers,
-      electionGreaters,
-      electionIndices
-    );
-    emit GovernanceSlashPerformed(account, penalty);
-    return true;
-  }
-
-  /**
-   * @notice Calls `LockedGold.slash` on `account` if `account` has an entry in `slashed`.
-   * @param account Account to slash
-   * @param electionLessers Lesser pointers for slashing locked election gold.
-   * @param electionGreaters Greater pointers for slashing locked election gold.
-   * @param electionIndices Indices of groups voted by slashed account.
-   */
-  function slashL2(
```

Comment:
I'd keep the slash function that calls `slash()` just to avoid breaking any tooling that's been built

Reply:
Not sure what you mean, since it was already flagged as `onlyL1` so tooling should already be breaking if they still exist.

Reply:
Sorry I misswrote, what I meant is to leave the function `slashL2(...)` not to have a breaking change, and leave the `slash(...)` one as another way to call `slashL2(...)`, so we can eventually deprecate `slashL2(...)`.